### PR TITLE
81 adapt recieve data (Dust)

### PR DESCRIPTION
**Pull request by fuzzyers**
Dust sensor post and read code is up

api/recieve-data {post}
api/dust-data {get}

### CODE REVIEW

File: fullstack/src/app/api/dust-data/route.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * @api {get} /api/dust-data Get dust data
+ * @description This file is the route for the dust-data API
+ */
+import db from "@/db";
+
+export const GET = async () => {
+    try {
+        const data = await db.any('SELECT * FROM dust');
```
Comment (SamanthaCra) [inline fullstack/src/app/api/dust-data/route.js:9]: Are we sure this needs to be a select all?

**Review by SamanthaCra — APPROVED** [review]
Is this updating the frontend at all?

File: fullstack/src/app/api/dust-data/route.js (hunk 1 above)
Comment (fuzzyers) [reply fullstack/src/app/api/dust-data/route.js:9]: We will probably want a variety of selects possibly including a select all for raw data pages.
But we will likely also want to have selects where the time is in a specific timeframe instead off all data

**Comment by fuzzyers** [comment]
> Is this updating the frontend at all?

Nope this is all backend

**Pull request merged**